src/app/shared/pipes/pipes.ts

```diff
@@ -17,3 +17,16 @@ export class DateCalcPipe implements PipeTransform {
     }
   }
 }
+
+@Pipe({ name: 'shorten' })
+export class ShortenLabelPipe implements PipeTransform {
```

Comment:
Can you add some documentation about this pipe, specially how does it shorten the label?

Reply:
Sure let me add it.